.travis.yml

```diff
@@ -18,7 +18,7 @@ before_script:
 - fluxbox &
 - wget http://selenium-release.storage.googleapis.com/2.44/selenium-server-standalone-2.44.0.jar
   -O selenium-server-standalone.jar
-- java -jar selenium-server-standalone.jar > /dev/null 2>/dev/null &
+- java -jar selenium-server-standalone.jar > selenium.log &
```

Comment:
I tried in local, and concluded that the output is going out only in one place, because when I tried this:

```
java -jar selenium-server-standalone.jar > selenium.log 2>selenium-errors.log &
```

there was no output at selenium.log and all the normal output went to selenium-errors.log

what do you think?